IonDotnet/Tree/IonSymbol.cs

```diff
@@ -5,6 +5,7 @@ namespace IonDotnet.Tree
     public sealed class IonSymbol : IonText
     {
         private int _sid;
+        private ImportLocation _importLocation;
 
         public IonSymbol(string text, int sid = SymbolToken.UnknownSid) : this(new SymbolToken(text, sid))
```

Comment:
SymbolToken object creation is missing importLocation parameter

Reply:
Added.